index.html

```diff
@@ -59,5 +59,218 @@ <h3>LET'S CONNECT</h3>
         </div>
       </div>
     </section>
+
+    <div class="grid-container">
+      <container class="grid-item">
+        <div class="item1">
+          <img src="/images/work1.jpg" alt="Work 1" />
+          <h1>Tonic</h1>
+        </div>
+
+        <div class="item2">
+          <p class="canopy">CANOPY</p>
+          <span class="point"></span>
+          <p>Back End Dev</p>
+          <span class="point"></span>
+          <p>2015</p>
+        </div>
+
+        <div class="item3">
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+        </div>
+
+        <div class="item4">
+          <ul>
+            <li>Html</li>
+            <li>Css</li>
+            <li>Javascript</li>
+          </ul>
+        </div>
+
+        <div class="item5">
+          <button>See project</button>
+        </div>
+      </container>
+
+      <container class="grid-item">
+        <div class="item1">
+          <img src="/images/work2.jpg" alt="Work 1" />
+          <h1>Multi-Post <br />Stories</h1>
+        </div>
```

Comment:
- [ ] **[OPTIONAL]** Please consider using `h2` and `h3` instead of `h1`for these different title. Each page should have only one `h1`. This will improve highly the accessibility of your website 👍 💯

Reply:
This issue has been resolved.